client/src/main/java/io/pravega/client/stream/EventStreamWriter.java

```diff
@@ -28,13 +29,13 @@
      * than to wrap this with custom retry logic.
      * 
      * @param event The event to be written to the stream (Null is disallowed)
-     * @return A future that will complete when the event has been durably stored on the configured number of
+     * @return A completableFuture that will complete when the event has been durably stored on the configured number of
```

Comment:
Minor: This line ending isn't aligned with the others, could you fix it, please?